knowledge-base/grid-scroll-to-selected-row.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Scroll to selected row
+description: How to scroll a grid to the selected row
+type: how-to
+page_title: Scroll to selected row
+slug: grid-kb-scroll-to-selected-row
+position: 
+tags: 
+ticketid: 1513767
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Grid for Blazor</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+I would like to pre-select a row when the page is loaded, and I would like to show the grid with that row scrolled to the top, or sorted to appear on top.
+
+## Solution
+You can get find a selected row in the grid markup by the `k-state-selected` class it has, and use a bit of JS to scroll to it - browsers provide the `scrollIntoView()` method for that. WIth virtualization you would have to find the position of that row and scroll to it through the `Skip` parameter of the grid, however, because it may not be rendered.
```

Comment:
```suggestion
You can find a selected row in the grid markup by the `k-state-selected` class it has, and use a bit of JS to scroll to it - browsers provide the `scrollIntoView()` method for that. WIth virtualization you would have to find the position of that row and scroll to it through the `Skip` parameter of the grid, however, because it may not be rendered.
```